### PR TITLE
fixed https://github.com/projectM-visualizer/projectm/issues/825

### PR DESCRIPTION
**Pull request by Kyborg2011**
After such a little changes a debug app on both my Android 13 devices started working smoothly...

photo_2024-08-03 21 57 03
photo_2024-08-03 21 56 55

### CODE REVIEW

File: src/libprojectM/MilkdropPreset/Shaders/PresetWarpVertexShaderGlsl330.vert [hunk 1]
```diff
@@ -16,8 +16,8 @@ precision mediump float;
 layout(location = 0) in vec2 vertex_position;
 layout(location = 1) in vec2 rad_ang;
 layout(location = 2) in vec4 transforms;
-layout(location = 3) in vec2 center;
-layout(location = 4) in vec2 distance;
+layout(location = 3) in vec2 center2;
```
Comment (revmischa) [inline src/libprojectM/MilkdropPreset/Shaders/PresetWarpVertexShaderGlsl330.vert:19]: Can you please explain why changing this fixes it?

Comment (kblaschke) [reply src/libprojectM/MilkdropPreset/Shaders/PresetWarpVertexShaderGlsl330.vert:19]: Same question. The only explanation I could come up with is that Android's shader Compiler somehow has `center` and `distance` as keywords.

I'd not postfix those variables with a 2 in the case, as there's clearly no 1 here. We could instead rather prefix the variables with `in_` or something, which would both fix the potential keyword issue and not make the names look weird.

Comment (kblaschke) [reply src/libprojectM/MilkdropPreset/Shaders/PresetWarpVertexShaderGlsl330.vert:19]: I guess renaming the distance variable should suffice, as I cannot find any keyword or function `center` in any GL standard.

Could you try that, keeping `center` as-is and for example rename `distance` to `warpDistance`?

**Comment by kblaschke** [comment]
I've changed the variable names according, keeping the commit info as-is otherwise.

Since we need to merge it into two branches (the fix will become part of the 4.1.2 release), there are now two new PRs open, #832 and #833. Closing this one here.

**Pull request closed without merging**